server/src/main/java/io/spine/server/commandbus/CommandBus.java

```diff
@@ -413,6 +448,32 @@ public Builder setThreadSpawnAllowed(boolean threadSpawnAllowed) {
             return this;
         }
 
+        /**
+         * Inject the {@link SystemGateway} of the bounded context to which the built bus belongs.
+         *
+         * <p>This method is {@link Internal} to the framework. The name of the method starts with
+         * {@code inject} prefix so that this method does not appear in an autocomplete hint for
+         * {@code set} prefix.
+         */
+        @Internal
+        public Builder injectSystemGateway(SystemGateway gateway) {
```

Comment:
How do you unit-test that the argument is injected? I'd have some tests similar to those that verify the correctness of the `Builder` implementation.